easybuild/easyconfigs/x/xtb-IFF/xtb-IFF-1.1-gfbf-2023a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'Tarball'
+
+name = 'xtb-IFF'
+version = '1.1'
+
+homepage = 'https://github.com/grimme-lab/xtbiff'
+description = """
+General Intermolecular Force Field based on Tight-Binding Quantum Chemical Calculations. 
+"""
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+source_urls = ['https://github.com/grimme-lab/xtbiff/releases/download/v1.1']
+sources = ['xtbiff.tar.xz']
+checksums = ['6d5dee10ba39ecb1e26374b7bf82e2364d8ebed5b6f35cdce4ced73245c296f5']
+
+dependencies = [
+    ('CREST', '2.12'),
```

Comment:
`CREST` is not a dep of `xtb-IFF`.
if there is any, it should be the other way around: `xtb-IFF` as a dep of `CREST`